### DISCUSSION TITLE
Join transformation with query selection

### DISCUSSION THREAD

**Discussion by luizluca** [grafana/grafana#57857] [General]
Today, join transformations merge all available queries into a single table. It would be nice if we could select which queries should be joined, keeping those unselected queries as they are. It would be even better if we could stack those join transformations, joining the result of a join transformation with another query, possibly using a different attribute.

**Comment by camsellem** [comment]
Upvoting this one

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
